libs/common/src/constants/ukef.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * A constant object containing UKEF (UK Export Finance) related information.
+ *
+ * @property {string} DOMAIN - The domain name for UK Export Finance.
+ */
+export const UKEF = {
+  DOMAIN: 'ukexportfinance.gov.uk',
+} as const;
```

Comment:
Nice - was thinking about something like this in another PR for a "UKEF" string. I think `UKEF.ACRONYM` would be most appropriate, also allows for a full name const.

Reply:
Have added, good shout.

Reply:
Thanks, I was going to do - it's currently not an acronym, should be `UKEF` 😛

Reply:
Have updated.

Reply:
Nice